src/transformers/generation/flax_utils.py

```diff
@@ -63,6 +63,7 @@ class FlaxGreedySearchOutput(ModelOutput):
     """
 
     sequences: jnp.ndarray = None
+    scores: jnp.ndarray = None
```

Comment:
Could we also update the docstring with the `scores` please?

Reply:
Of Course! Added.

Reply:
Thank you!